three/three.h

```diff
@@ -5,27 +5,66 @@
 #ifndef DATADOG_AGENT_SIX_THREE_H
 #define DATADOG_AGENT_SIX_THREE_H
 #include <map>
+#include <mutex>
 #include <string>
 #include <vector>
 
 #include <Python.h>
 #include <six.h>
 
+// we only populate the fields `m_base` and `m_name`, we don't need any of the
+// rest since we're doing Single-phase initialization
+//
+// INIT_PYTHON_MODULE creates the def_<moduleName> (a PyModuleDef struct) and
+// the needed PyInit_<moduleName> callback.
+#define INIT_PYTHON_MODULE(moduleID, moduleName)                                                                       \
```

Comment:
Since they're not used anywhere else, can we move macro defintions in the cpp module?

Reply:
:+1: